static/etc/system-image/writable-paths

```diff
@@ -69,3 +69,4 @@
 /var/lib/gdm3                           auto                    persistent  transition  none
 /usr/share/wayland-sessions             auto                    persistent  transition  none
 /usr/share/polkit-1                     auto                    persistent  transition  none
+/var/lib/colord                         auto                    persistent  transition  none
```

Comment:
/var/lib/fprintd...?

Reply:
I forgot to push, James had added the path for colord as well